presto-docs/src/main/sphinx/release/release-0.276.rst

```diff
@@ -0,0 +1,53 @@
+=============
+Release 0.276
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug where a query with an order by clause and an unpartitioned window function sometimes returned unordered results.
+* Improve Graphviz output for JOIN nodes and estimate stats.
+* Reduce the number of hdfsConfiguration copies in the worker. This feature is enabled by default and can be controlled by setting the ``hive.copy-on-first-write-configuration`` configuration property appropriately.
+* Add sized-based split weights for Hudi connector to improve query performance. This is enabled by default, controlled by configuration property ``hudi.size-based-split-weights-enabled`` and session property ``hudi.size_based_split_weights_enabled``. Two more configuration properties are added to adjust the weight: ``hudi.standard-split-weight-size`` to configure the split size corresponding to the standard split weight, and ``hudi.minimum-assigned-split-weight`` to configure the minimum split weight.
+* Add support for complex JsonPath expressions in :func:`JSON_EXTRACT`, :func:`JSON_EXTRACT_SCALAR` and :func:`JSON_SIZE` using Jayway.
+* Add support to nested SQL functions with lambdas when ``inline_sql_functions = false``.
+* Add the types and SqlFunctions defined by GeoPlugin to `BuiltInTypeAndFunctionNamespaceManager.java` for registering.
+* Add documentation for :doc:`construct_tdigest`.
```

Comment:
This might not link properly to `construct_tdigest`.
From the commit it looks like it is inside tdigest documentation.
Try this instead :doc:\`/functions/tdigest\`